packages/react/src/ReferenceRangeEditor/ReferenceRangeEditor.test.tsx

```diff
@@ -6,17 +6,20 @@ import { MemoryRouter } from 'react-router-dom';
 import { MedplumProvider } from '../MedplumProvider/MedplumProvider';
 import { ReferenceRangeEditor, ReferenceRangeEditorProps } from './ReferenceRangeEditor';
 import { HDLDefinition, TestosteroneDefinition } from '../stories/referenceLab';
+import { deepClone } from '@medplum/core';
 
 const medplum = new MockClient();
 
 async function setup(args: ReferenceRangeEditorProps): Promise<void> {
   await act(async () => {
     render(
-      <MemoryRouter>
-        <MedplumProvider medplum={medplum}>
-          <ReferenceRangeEditor {...args} />
-        </MedplumProvider>
-      </MemoryRouter>
+      <React.StrictMode>
```

Comment:
Out of curiosity, why add `<React.StrictMode>` here?

Reply:
I ran into some bugs on app.medplum.com that were surfaced by StrictMode, caused by double calling `setState`. I wanted to make sure I would start surfacing those kinds of issues in my tests. 

Was orthogonal to this specific bug, but thought it would be better to add it in for future features